client/src/components/Lists/PeersHealth.js

```diff
@@ -6,40 +6,87 @@ import React from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import ReactTable from '../Styled/Table';
 import { peerStatusType } from '../types';
+import Tooltip from "@mui/material/Tooltip";
+import styled from "@emotion/styled";
```

Comment:
Q: are these libraries already added to project?

Reply:
No, earlier it was not there

Reply:
these are not added in the package.json

Reply:
Added to package.json file with required libraries .